### ISSUE TITLE
A way to disable _source?

### ISSUE THREAD

**Issue by negon** [ongr-io/ElasticsearchBundle#706]
Using attachments I would like to disable the _source field, but there is no _source meta-field?

Reference: https://www.elastic.co/guide/en/elasticsearch/reference/2.4/mapping-source-field.html

Thanks :)

**Comment by saimaz** [comment]
It can be disabled by options. This parameter was introduced for passing any required options to the type.

I'm going to introduce `options` in document annotation where you can set anything to type mapping. It will be released in `v1.2.5`. 

Usage:
```php
/**
 * @ES\Document(type="attachment", options={"_source":{"enabled":"false"}})
 */
class Attachment
{
//...
```

**Comment by negon** [comment]
Thank you very much! Must check more thoroughly next time :+1:
